packages/cli/src/commands/generate/sdl/templates/sdl.js.template

```diff
@@ -37,6 +37,15 @@ export const schema = gql`
     ${createInput}
   }
 
+  <% if (idInput) { %>
+    <% if (docs) { %>
+      """Autogenerated ID input type of ${singularPascalName}Id."""
+    <% } %>
+    input ${singularPascalName}IdInput {
+      ${idInput}
+    }
+  <% } %>
```

Comment:
curious... why do we duplicate the templates? might make more sense to always generate typescript, then transpile to js when not in ts mode